### PR TITLE
LPS-192966 Use Experience Selector in the Simulation panel

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPS-192966

Just check the commits from 633a8643d54e581dd53ba642766f484cabf92c47 to ec2832cdc07651c9adfc16c088c5fa7c2300032b. The rest of the commits belong to https://github.com/liferay-echo/liferay-portal/pull/13121

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Hi @veroglez , 

the simulation by segment is not working and I believe is because we deleted the input element for segment entries. Also, the design is a bit different now (we have the active tick and the left side of the option) and that makes the secondary text break into two lines all the times. I don't know if we could improve this ins any way:

Before:
Captura de pantalla 2023-08-09 a las 10 48 19

After:
Captura de pantalla 2023-08-09 a las 10 43 23

File: modules/apps/segments/segments-simulation-web/src/main/resources/META-INF/resources/js/components/SegmentSelector.tsx [hunk 1]
```diff
@@ -3,109 +3,77 @@
...
 				<p>{Liferay.Language.get('no-segments-have-been-added-yet')}</p>
 			) : (
-				<div className="form-group">
-					<label htmlFor={`${namespace}segmentsEntryId`}>
+				<Form.Group>
+					<label htmlFor={selectorId} id={labelId}>
 						{Liferay.Language.get('segment')}
 					</label>
 
-					<input
```
[the remark this answers is not in the document]
Comment (veroglez) [reply modules/apps/segments/segments-simulation-web/src/main/resources/META-INF/resources/js/components/SegmentSelector.tsx:46]: Yesss, my mistake! I update it!

**Comment by veroglez** [comment]
> Hi @veroglez ,
> 
> the simulation by segment is not working and I believe is because we deleted the input element for segment entries. Also, the design is a bit different now (we have the active tick and the left side of the option) and that makes the secondary text break into two lines all the times. I don't know if we could improve this ins any way:
> 
> Before: Captura de pantalla 2023-08-09 a las 10 48 19
> 
> After: Captura de pantalla 2023-08-09 a las 10 43 23

-------------
Yes, it looks like Clay's Picker component has more padding than the previous one (image from its storybook).
Screen Shot 2023-08-09 at 1 24 18 PM

I see you also have the scroll bar activated by default on the Mac and that takes up space. If we disable it it looks like this:

Screen Shot 2023-08-09 at 1 37 07 PM

Maybe one option would be to talk to design to be able to distribute the elements in another way.

**Comment by claraizquierdo** [comment]
> > Hi @veroglez ,
> > the simulation by segment is not working and I believe is because we deleted the input element for segment entries. Also, the design is a bit different now (we have the active tick and the left side of the option) and that makes the secondary text break into two lines all the times. I don't know if we could improve this ins any way:
> > Before: Captura de pantalla 2023-08-09 a las 10 48 19
> > After: Captura de pantalla 2023-08-09 a las 10 43 23
> 
> Yes, it looks like Clay's Picker component has more padding than the previous one (image from its storybook). Screen Shot 2023-08-09 at 1 24 18 PM
> 
> I see you also have the scroll bar activated by default on the Mac and that takes up space. If we disable it it looks like this:
> 
> Screen Shot 2023-08-09 at 1 37 07 PM
> Maybe one option would be to talk to design to be able to distribute the elements in another way.

Ok @veroglez , we can change this later if the design team decides it.

**Review by claraizquierdo — APPROVED** [review]
🎉

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by liulukiwi** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
Hi @veroglez, should LPS-191512 or LPS-192591?

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/13239

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**